pkg/userdata/centos/provider_test.go

```diff
@@ -193,6 +193,16 @@ func TestUserDataGeneration(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "kubelet-v1.21-nutanix",
+			spec: clusterv1alpha1.MachineSpec{
+				ObjectMeta: metav1.ObjectMeta{Name: "node1"},
+				Versions: clusterv1alpha1.MachineVersionInfo{
+					Kubelet: "1.21.5",
+				},
+			},
+			cloudProviderName: stringPtr("nutanix"),
```

Comment:
Nit: should/could we use https://github.com/kubermatic/machine-controller/blob/17474051fe67702a33a453ff1883b6008983988f/pkg/providerconfig/types/types.go#L56 here instead?

Reply:
Seems to be done that way for other test cases too. Resolving.